src/storage/txn/scheduler.rs

```diff
@@ -786,6 +795,18 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
         };
         SCHED_STAGE_COUNTER_VEC.get(tag).write.inc();
 
+        let quota_delay = quota_limiter.consume_write(write_bytes, cost_time);
+        if !quota_delay.is_zero() {
+            TXN_COMMAND_THROTTLE_TIME_COUNTER_VEC_STATIC
+                .get(tag)
+                .inc_by(quota_delay.as_micros() as u64);
+            GLOBAL_TIMER_HANDLE
+                .delay(std::time::Instant::now() + quota_delay)
+                .compat()
+                .await
+                .unwrap();
+        }
```

Comment:
This looks not so accurate.
1. The counted written bytes will be larger than actual because no data will be written if a pessimistic lock is encountered or pessimistic locks are written only into the memory.
2. The cpu time is not accurate either. If an error is returned in L786, it is not counted.

Both look not very important, but better to leave a comment here.

Reply:
For question 2, most of cpu resource will be used in the above block, so it's doesn't matter.

![image](https://user-images.githubusercontent.com/18193301/158523787-0a225609-5596-4fdb-9998-c19c4c20fa6c.png)